src/js/tracks/text-track-settings.js

```diff
@@ -235,113 +235,112 @@ function setSelectedOption(target, value) {
-              <option value="#0FF">Cyan</option>
-            </select>
-            <span class="vjs-text-opacity vjs-opacity">
-              <select>
-                <option value="">---</option>
-                <option value="1">Opaque</option>
-                <option value="0.5">Semi-Opaque</option>
+         <div class="vjs-tracksettings-colors">
+          <fieldset>
+            <legend>Colors</legend>
+            <div class="vjs-fg-color vjs-tracksetting">
+              <label class="vjs-label" for="captions-foreground">Foreground</label>
+              <select id="captions-foreground">
+                <option value="#FFF" selected>White</option>
+                <option value="#000">Black</option>
+                <option value="#F00">Red</option>
+                <option value="#0F0">Green</option>
+                <option value="#00F">Blue</option>
+                <option value="#FF0">Yellow</option>
+                <option value="#F0F">Magenta</option>
+                <option value="#0FF">Cyan</option>
               </select>
-            </span>
-        </div> <!-- vjs-fg-color -->
-        <div class="vjs-bg-color vjs-tracksetting">
-            <label class="vjs-label">Background</label>
-            <select>
-              <option value="">---</option>
-              <option value="#FFF">White</option>
-              <option value="#000">Black</option>
-              <option value="#F00">Red</option>
-              <option value="#0F0">Green</option>
-              <option value="#00F">Blue</option>
-              <option value="#FF0">Yellow</option>
-              <option value="#F0F">Magenta</option>
-              <option value="#0FF">Cyan</option>
-            </select>
-            <span class="vjs-bg-opacity vjs-opacity">
+              <span class="vjs-text-opacity vjs-opacity">
+                <select>
+                  <option value="1" selected>Opaque</option>
+                  <option value="0.5">Semi-Opaque</option>
+                </select>
+              </span>
+            </div>
+            <div class="vjs-bg-color vjs-tracksetting">
+              <label class="vjs-label" for="captions-background">Background</label>
+              <select id="captions-background">
+                <option value="#000" selected>Black</option>
+                <option value="#FFF">White</option>
+                <option value="#F00">Red</option>
+                <option value="#0F0">Green</option>
+                <option value="#00F">Blue</option>
+                <option value="#FF0">Yellow</option>
+                <option value="#F0F">Magenta</option>
+                <option value="#0FF">Cyan</option>
+              </select>
+              <span class="vjs-bg-opacity vjs-opacity">
                 <select>
-                  <option value="">---</option>
                   <option value="1">Opaque</option>
+                  <option value="0.8" selected>80% Opaque</option>
```

Comment:
@gkatsev this is an FCC requirement (FCC 12-9), not a WCAG requirement. Specifically, it calls for:

> four settings, opaque (100% opacity), semi-transparent (at 75% or 25% opacity), and transparent (0% opacity)

for `background` and for `window`, and three for `font` (hmm, actually transparent font is essentially what we need for description tracks! That gives me an idea!)

So yes, we need to add another `semi-transparent`. Whether we need to change the values (to 0.75 and 0.25) is something to discuss, too.